cmd/bnscli/clitests/submit_send_tokens.test.gold

```diff
@@ -0,0 +1 @@
+exit code: 0
```

Comment:
Nice, so this one did run, while the sign_send one did not?
What did you run against?

Also interesting that this is an unsigned transaction and would be rejected by CheckTx, yet `bnscli submit` considered it a success. So I think there is an issue with `bnscli submit`

Reply:
The `bnscli` does not validate content but relies on the server to do the work.
In this case, I am running a fake server so that the result is reproducible. The server does not validate anything and does not care about the requested content. It always responds with a successful message for the requested HTTP path. That's why submitting an unsigned request succeeds.

I try to keep the tests minimal. This test is only about submitting a message. There is no great way here to ensure that submitted content is correct, only that the command works as expected (pipe a transaction to it and it will be send). The implementation details are tested in the Go unit test though.

Does this make sense?